### ISSUE TITLE
[GSSoC'23]: Dinosaur Game

### ISSUE THREAD

**Issue by singh-anushka** [ssitvit/Games-and-Go#111]
In this game, there is a prehistoric landscape where our player is a dinosaur. The objective is simple: help the dinosaur navigate through an endless desert while avoiding obstacles and surviving for as long as possible. As you progress, the game becomes increasingly challenging. Cacti and flying pterodactyls will try to block your path, and you must time your jumps and crouches perfectly to avoid them.

Kindly assign me this issue.

**Comment by raj03kumar** [comment]
You already have 3 issue. Are you sure you can complete it alongside.

**Comment by singh-anushka** [comment]
Yes, I can. I adding the issue for which I'm already working on.
